src/languageProvider/locales/en_US.json

```diff
@@ -350,22 +354,12 @@
   "tutorial6.title": "Decentralization",
   "tx.approveBotTransfer": "Approve BOT Transfer",
   "tx.resetApproval": "Reset Approval",
-  "txConfirm.message": "You are about to {txDesc} for {txAmount} {txToken}. Please click OK to continue.",
-  "txConfirm.messageWithFee": "You are about to {txDesc} for {txAmount} {txToken} with a maximum transaction fee of {txFee} QTUM. Any unused transaction fees will be refunded to you. Please click OK to continue.",
+  "txConfirm.approveNotice": "The {action} action requires you to approve BOT to be transferred first. You will be required to confirm the follow-up {action} transaction after this transaction is successful.",
```

Comment:
for other language

Reply:
I will run the lang script when this review is done. Then we can add it in.